utils.groovy

```diff
@@ -396,4 +396,29 @@ def get_cosa_img(pipecfg, stream) {
     return utils.substituteStr(cosa_img, [STREAM: stream])
 }
 
+// Run a closure inside a context that has access to the AWS Build
+// Upload credentials.
+def withAWSBuildUploadCredentials(Closure body) {
+    withCredentials([file(variable: 'AWS_CONFIG_FILE',
```

Comment:
Shouldn't this be `AWS_BUILD_UPLOAD_CONFIG`?

Reply:
yep!

Reply:
fixed!